### ISSUE TITLE
Action fails without error message

### ISSUE THREAD

**Issue by MartinX3** [repo-sync/github-sync#28]
Out of nowhere without me changing the script, the run just fails. (the `omnirom-9.0-sync` one)
Is there a way to debug it to find the reason?
That's my repo: https://github.com/MartinX3-AndroidDevelopment/android_bootable_recovery

Config:
```
jobs:
  repo-sync:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@master
    - name: upstream-sync
      uses: repo-sync/github-sync@v2
      with:
        source_repo: "TeamWin/android_bootable_recovery"
        source_branch: "*"
        destination_branch: "*"
        github_token: ${{ secrets.GITHUB_TOKEN }}
    - name: omnirom-9.0-sync
      uses: repo-sync/github-sync@v2
      with:
        source_repo: "oshmoun/android_bootable_recovery"
        source_branch: "android-9.0"
        destination_branch: "omnirom/android-9.0"
        github_token: ${{ secrets.GITHUB_TOKEN }}
```

Log part:
```2020-08-24T22:11:03.0495266Z ##[group]Run repo-sync/github-sync@v2
2020-08-24T22:11:03.0495419Z with:
2020-08-24T22:11:03.0495561Z   source_repo: oshmoun/android_bootable_recovery
2020-08-24T22:11:03.0495681Z   source_branch: android-9.0
2020-08-24T22:11:03.0495829Z   destination_branch: omnirom/android-9.0
2020-08-24T22:11:03.0496801Z   github_token: ***
2020-08-24T22:11:03.0496933Z ##[endgroup]
2020-08-24T22:11:03.0512163Z ##[command]/usr/bin/docker run --name b3ac6c82b98c513594016a497696aa8f40ee8_ed8a2f --label 3b3ac6 --workdir /github/workspace --rm -e INPUT_SOURCE_REPO -e INPUT_SOURCE_BRANCH -e INPUT_DESTINATION_BRANCH -e INPUT_GITHUB_TOKEN -e GITHUB_TOKEN -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/android_bootable_recovery/android_bootable_recovery":"/github/workspace" 3b3ac6:c82b98c513594016a497696aa8f40ee8  "oshmoun/android_bootable_recovery" "android-9.0:omnirom/android-9.0"
2020-08-24T22:11:03.3673740Z UPSTREAM_REPO=https://github.com/oshmoun/android_bootable_recovery.git
2020-08-24T22:11:03.3681075Z BRANCHES=android-9.0:omnirom/android-9.0
```
![grafik](https://user-images.githubusercontent.com/4518092/91101906-5da05f80-e668-11ea-9196-e3215b97ae0c.png)

**Comment by wei** [comment]
We recently pointed `v2` to `v2.1.0` which includes fix for #21. It has caused an issue when multiple `repo-sync/github-sync` actions are stacked. I have added the fix in 5e864327cffda5b18a5301b851a4169c69aa48df and more logging 32f5a7606cf2ae8f915bacaadb50c237f0a075d1 . The latest `v2` tag is now pointing to `v2.1.2`

It seems issues with `repo-sync/github-sync` step in your workflow has been resolved. Please let us know if you encounter any more issues. Thank you!

**Comment by MartinX3** [comment]
Woah, that was fast!
Thank you very much! :)
